webpack.config.js

```diff
@@ -0,0 +1,63 @@
+const path = require('path');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
+
+
+module.exports = (env, argv) => {
+  const config = {
+    entry: {'react-streamfield': ['./src/index.js', './src/index.scss']},
+    output: {
+      path: path.resolve('dist'),
+      filename: '[name].js',
+      publicPath: '/static/'
+    },
+    module: {
+      rules: [
+        {
+          test: /\.jsx?$/,
+          loader: 'babel-loader',
+          exclude: /node_modules/,
+        },
+        {
+          test: /\.scss$/,
+          use: [
+            MiniCssExtractPlugin.loader,
```

Comment:
It's mandatory for a project like this to use [Autoprefixer](https://github.com/postcss/autoprefixer).